spec/std/string_pool_spec.cr

```diff
@@ -65,4 +65,12 @@ describe StringPool do
     end
     pool.size.should eq(10_000)
   end
+
+  it "can be created with larger initial capacity" do
```

Comment:
It is smaller with the other change, though.

Reply:
hehe, yes, but reverted back to 8 entries now, so now it's true again ;)